bahmni_atom_feed/models/atom_event_worker.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, api
+import logging
+import json
+import uuid
+
+STATE_CODE_PREFIX = 'UNKNOWN-'
+_logger = logging.getLogger(__name__)
+
+
+class AtomEventWorker(models.Model):
```

Comment:
There is no Atom Event. Can be confusing for people reading this code.. they think this code is parsing an AtomFeed Event.. which is not the case. This is just a Bahmni event with Customer Data. Can we rename this?

Reply:
class AtomEventWorker was renamed to ApiEventWorker for better clarity and understanding.